python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -898,6 +910,41 @@ def _fast_slow_function_call(func: Callable, /, *args, **kwargs) -> Any:
                 # try slow path
                 raise Exception()
             fast = True
+
+            if get_option(debug_mode) | _env_get_bool(
+                "CUDF_PANDAS_DEBUG", False
+            ):
+                try:
+                    with nvtx.annotate(
+                        "EXECUTE_SLOW",
+                        color=_CUDF_PANDAS_NVTX_COLORS["EXECUTE_SLOW"],
+                        domain="cudf_pandas",
+                    ):
+                        slow_args, slow_kwargs = (
+                            _slow_arg(args),
+                            _slow_arg(kwargs),
+                        )
+                        with disable_module_accelerator():
+                            slow_result = func(*slow_args, **slow_kwargs)
+                except Exception as e:
+                    warnings.warn(
+                        "The result from pandas could not be computed correctly. "
```

Comment:
I wish there was a way to write something like `{slow_module_name}` instead of `pandas` but I don't see an immediate solution for that.
```suggestion
                        "The result from pandas could not be computed. "
```